lib/inspec/schema.rb

```diff
@@ -84,6 +84,7 @@ class Schema
         'id' => { 'type' => 'string' },
         'title' => { 'type' => %w{string null} },
         'desc' => { 'type' => %w{string null} },
+        'descriptions' => { 'type' => %w{array} },
```

Comment:
I agree `desc_sections` follows more naturally. `desc_sections.each do |section|`

Reply:
I think the reason behind `descriptions` is that we want to deprecate `desc`(schema output) and eventually remove it. I agree though during transition it will be a bit confusing.